go/service/main.go

```diff
@@ -227,6 +233,33 @@ func (d *Service) checkTrackingEveryHour() {
 	}()
 }
 
+func (d *Service) gregordConnect() error {
+	d.G().Log.Debug("gregor URI: %s", d.G().Env.GetGregorURI())
+	h := newGregorHandler(d.G())
+	if d.G().Env.GetGregorUseTLS() {
+		return d.gregordConnectTLS(h)
+	}
+	return d.gregordConnectNoTLS(h)
+}
+
+func (d *Service) gregordConnectTLS(h *gregorHandler) error {
+	d.G().Log.Debug("connecting to gregor via TLS")
+	rawCA := d.G().Env.GetBundledCA(d.G().Env.GetGregorURI())
```

Comment:
Can you change the `ca.go` file to make the gregor host `gregor.api.keybase.io`, and then use the same CA for both?